lib/newspaper_works/text_extraction/render_alto.rb

```diff
@@ -39,13 +40,20 @@ def alto_page(pxwidth, pxheight, &block)
           builder
         end
 
+        def scale_point(value)
+          # note: presuming non-fractional, even though ALTO 2.1
+          #   specifies coordinates are xsd:float, not xsd:int,
+          #   simplify to integer value for output:
+          (value * @scaling).to_i
+        end
+
         # return layout for page
         def alto_layout(xml, pxwidth, pxheight, &block)
           xml.Layout do
             xml.Page(ID: 'ID1',
                      PHYSICAL_IMG_NR: '1',
                      HEIGHT: pxwidth.to_i,
-                     WIDTH: pxwidth.to_i) do
+                     WIDTH: pxheight.to_i) do
```

Comment:
Just checking, this is correct, yes? `HEIGHT: pxwidth`, `WIDTH: pxheight`

(It worked fine in the UI, just looks a bit weird.)

Reply:
AFAICT, I think I accidentally got these identifiers switched, even though the array order in the end outcome JSON is correct.  This needs correcting, or the code remains confusing.  I will correct ASAP.

Reply:
Ok, strike last comment ^^^.  This is still wrong, but I was confusing with ALTO reading, not ALTO generation.  This smells like a bug, but scope would be limited to downloaded ALTO, not JSON word coordinates (which are created from OCR, not from this generated ALTO).

Reply:
Corrected in 9ad9308e8cad7753c5c935ac5ed098337cc46fe3